FrontEnd/main/src/components/landing-page/partners/Partners.js

```diff
@@ -0,0 +1,33 @@
+import './Partners.css';
+
+const MainPartners = () => {
+    return (
+        <div className="partners-section">
+          <div className="partners-section__header">
+              <div className="partners-section__header-text">партнери</div>
```

Comment:
For the future, having file LocalizedText.js (or some other name) with all this text, that is displayed for the user would be great

Reply:
Thanks